### PR TITLE
fix: allow attendee checkin for several days

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/2016

require https://github.com/fossasia/open-event-server/pull/8021

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by mariobehling** [comment]
Please add screenshots to all PRs.

**Comment by maze-runnar** [comment]
> Please add screenshots to all PRs.

there is nothing to as a screenshot in this PR, it can be tested only.

**Review by MeghalBisht — DISMISSED** [review]

**Review by ranjitkshah — APPROVED** [review]

**Review by sachinchauhan2889 — COMMENTED** [review]
getting status 422.
Am i doing anything wrong???

File: app/controllers/events/view/tickets/attendees/list.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
     const attendee = this.store.peekRecord('attendee', attendee_id, { backgroundReload: false });
     attendee.toggleProperty('isCheckedIn');
     if (attendee.isCheckedIn) {
-      const newCheckinTimes = attendee.get('checkinTimes') === null ? `${moment().toISOString()}` : `${attendee.get('checkinTimes')},${moment().toISOString()}`;
+      const newCheckinTimes = attendee.get('checkinTimes') === null ? `${moment().toISOString()}` : `${moment().toISOString()}`;
```
Comment (sachinchauhan2889) [inline app/controllers/events/view/tickets/attendees/list.js:77]: Server is already merging times.

**Review by sachinchauhan2889 — DISMISSED** [review]

**Review by sachinchauhan2889 — DISMISSED** [review]

**Review by ranjitkshah — APPROVED** [review]

**Comment by mariobehling** [comment]
Please add screenshots.

**Review by MeghalBisht — APPROVED** [review]

**Comment by maze-runnar** [comment]
> Please add screenshots.


https://user-images.githubusercontent.com/56407566/121459032-223d0a80-c9c8-11eb-8839-cf15dfbd1a2b.mp4

**Pull request closed without merging**